package.json

```diff
@@ -46,8 +47,10 @@
     "eslint-plugin-jsx-a11y": "^6.1.1",
     "eslint-plugin-promise": "^4.0.0",
     "eslint-plugin-react": "^7.11.1",
+    "eslint-plugin-react-hooks": "^1.2.0",
     "eslint-plugin-unicorn": "^7.1.0",
-    "babel-eslint": "^10.0.1"
+    "react": "^16.8.3",
+    "react-dom": "^16.8.3"
```

Comment:
are `react` and `react-dom` hard dependencies?

Reply:
No they aren’t, I had installed them without saving but it looks like they still got added in there. Going to remove those.